src/client.rs

```diff
@@ -38,11 +38,27 @@ pub trait HttpRoundTripper {
     /// The type for errors generated by the roundtripper.
     type Err: error::Error;
 
-    /// Make an HTTP request.  In practice only POST request will be made.
+    /// Make a synchronous HTTP request. In practice only POST request will be made.
+    #[cfg(not(feature = "async"))]
```

Comment:
Just skimming, but if I understand correctly, the following would be my recommendations:

A PR like that should name async functions differently, and keep the sync versions available even when the feature is enabled. Otherwise multiple libraries might pull in `jsonrpc`, and then if one uses async it would break the one that doesn't.

And despite that - just because I need `async` in one place of the code, doesn't mean I won't need a sync version in other.

Reply:
Good points! I've updated the PR, although there is a bit of code duplication now.